internal/gui/time.go

```diff
@@ -18,34 +16,31 @@ func newTimeWindow(g *Gui) (t *timeWindow) {
 		Table: tview.NewTable().SetSelectable(false, false).SetFixed(3,1),
 	}
 
-	t.SetBorder(true)
-	t.setEntries(g)
-	t.setKeybinding(g)
+	t.SetBorder(false)
+	t.setEntries()
+	//t.setKeybinding(g)
```

Comment:
remove it ?

Reply:
Actually since the date field (`time.go`) is not being used currently in the layout I will remove the file.